bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java

```diff
@@ -180,6 +180,11 @@ public static short getFlags(int packetHeader) {
      */
     int ETOOMANYREQUESTS = 106;
 
+    /**
+     * Ledger in unknown state.
+     */
+    int EUNKNOWN = 107;
```

Comment:
do we need to bump up CURRENT_PROTOCOL_VERSION?

Reply:
What are the current guidelines for modifying the CURRENT_PROTOCOL_VERSION?

Reply:
I don't think we have it documented (@sijie and @merlimat can correct me) but the rule of thumb was around old client not handling new responses correctly/doing something dangerously wrong when it gets the new response.